generator/gen_const.go

```diff
@@ -132,6 +132,11 @@ func (gen *Generator) expandEnum(wr io.Writer, decl *tl.CDecl, namesSeen map[str
 	spec := decl.Spec.(*tl.CEnumSpec)
 	tagName := gen.tr.TransformName(tl.TargetType, decl.Spec.GetBase())
 	enumType := gen.tr.TranslateSpec(&spec.Type)
+	if tip, ok := gen.tr.TypeTipRx(tl.TipScopeEnum, string(tagName)); ok {
+		if tip.TipAt(0) == tl.TipTypeUnsigned {
```

Comment:
Shouldn't this iterate over all the tips in the returned TipSpecRx? I'm not sure I'm understanding the code correctly, but it looks like multiple tips are possible for a single type name.

Also: this block should probably be duplicated in the function above for anonymous enums.

Reply:
I'll look again at doing this. I didn't see a good way to iterate through all of the tips, due to the slice being a private portion of the interface.

Will also consider doing this for "anonymous" enums (but it was unclear to me if this change should apply to those, since I assumed they didn't have names and could thus not be easily identified).

Reply:
Done and done. Thanks for the review.